editor.planx.uk/src/pages/FlowEditor/components/Submissions/components/FormattedResponse.tsx

```diff
@@ -38,7 +40,7 @@ export const FormattedResponse: React.FC<FormattedResponseProps> = ({
         <ReactJson
           src={parsedResponse}
           theme="monokai"
-          collapsed={2}
+          collapsed={expandAllLevels ? false : 2}
```

Comment:
Prop names are a little confusing here - but this allows us to configure a variant of the "console" that expands-all-levels (useful for small "example" output), rather than only expanding 2-levels by default (still desirable in sidebar).